code/034/script.js

```diff
@@ -0,0 +1,20 @@
+let person = {
```

Comment:
Please format the code indentation properly. 

Developer Tips: You can use [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) on vscode to auto format your code.

```diff
- let person = {
-     name: "John",
-     age: 30,
-     hobbies: ["reading", "traveling", "cooking"],
-     address: {
-       street: "123 Main St",
-       city: "New York",
-       state: "NY"
-     },
-     sayHello: function() {
-       console.log("Hello, my name is " + this.name);
-     }
-   };
-   
-   console.log(person.name); 
-   console.log(person.hobbies[1]); 
-   console.log(person.address.city); 
-   
-  
-   person.sayHello(); 

+ let person = {
+   name: "John",
+   age: 30,
+   hobbies: ["reading", "traveling", "cooking"],
+   address: {
+     street: "123 Main St",
+     city: "New York",
+     state: "NY",
+   },
+   sayHello: function () {
+     console.log("Hello, my name is " + this.name);
+   },
+ };
+ 
+ console.log(person.name);
+ console.log(person.hobbies[1]);
+ console.log(person.address.city);
+ 
+ person.sayHello();
```

Once formatted on code, please format the same on README file too